src/Routes.tsx

```diff
@@ -17,6 +18,7 @@ export const Routes: ReactNode = () => (
           <Route path="/oops" component={OopsPage} />
           <Route path="/no-permissions" component={NoPermissionsPage} />
           <Route path="/:SKU" component={DetailsPage} />
+          <Route path="/not-found" component={NotFoundPage} />
```

Comment:
Can't tell from the AC of the story - but I would imagine this wouldn't redirect to a not found page (wouldn't that break hitting the back button in the browser?). I figured it would just render the NotFoundPage at the same route

Reply:
After running it locally - it doesn't look like this route it actually used anywhere